client/src/components/App/Content/LandingPage/adminLandingPage/adminInvestigationsTable/adminInvestigationsTable.tsx

```diff
@@ -0,0 +1,183 @@
+                         </Tooltip>);
+            default:
+                return get(adminInvestigation, cellName);
+        }
+    };
+
+    const getProgressLength = (hours : number) => {
+        let props = {hours:hours, progressLen: 100, color: "red"};
+
+        if(hours <= 2) {
+            props.progressLen = 25;
+            props.color = "green";
+        } else if (hours <= 3) {
+            props.progressLen = 50;
+            props.color = "yellow";
+        } else if (hours <= 4) {
+            props.progressLen = 75;
+            props.color = "orange";
+        }
+
+        return props;
+    }
+
+    const onInvestigationRowClick = async (adminInvestigation: adminInvestigation) => {
+        const getComplexityReasonClickLogger = logger.setupVerbose({
+            workflow: 'get Complexity Reason when opening an investigation',
+            investigation: adminInvestigation.id,
+            user: user.id
+        });
+        await axios.get('/investigationInfo/getComplexityReason/'+ adminInvestigation.id)
+            .then((result) => {
+                if (result?.data && result.headers['content-type'].includes('application/json')) {
+                    setComplexReasonsId(result.data)
+                    getComplexityReasonClickLogger.info('the chosen investigation have complexity reasons', Severity.LOW);
+                } else { 
+                    setComplexReasonsId([]) 
+                    getComplexityReasonClickLogger.info('the chosen investigation dont have complexity reasons', Severity.LOW);
+                    }
+            })
+            .catch((errorMessage) => { getComplexityReasonClickLogger.error(errorMessage, Severity.HIGH); })
+        setInvestigationStatus({
+            mainStatus: adminInvestigation.investigation_status,
+            subStatus: adminInvestigation.sub_status,
+            statusReason: adminInvestigation.status_reason
+        });
+
+        moveToTheInvestigationForm(adminInvestigation.id);
+    };
+
+    const moveToTheInvestigationForm = async (epidemiologyNumberVal: number) => {
+        setLastOpenedEpidemiologyNum(epidemiologyNumberVal);
+        await persistor.flush();
+        window.open(investigationURL);
+    };
+
+    return (
+        <LoadingCard isLoading={isLoading} height={cardHeight}>
+            <TableContainer className={classes.tableStyle} component={Paper}>
+                <Table stickyHeader>
+                    <TableHead id='investigators-table-header'>
+                        <TableRow>
+                            {
+                                Object.keys(TableHeaders).map((cellName: string) => {
+                                    return (
+                                        <TableCell key={cellName}
+                                            sortDirection={orderBy === cellName ? order : false}
+                                        >
+                                            <Tooltip title={''}>
```

Comment:
Why the tooltip?

Reply:
yea I thought I might need it later, deleted